docs/source/readers.rst

```diff
@@ -128,6 +128,7 @@ Options                             Definition
 ``--adobe-2-0-metric``              Metric to include in the report
 ``--adobe-2-0-start-date``          Start date of the period to request (format: YYYY-MM-DD)
 ``--adobe-2-0-end-date``            Start date of the period to request (format: YYYY-MM-DD)
+``--adobe-2-0-date-range``          Date range. By default, not available in Adobe, so choose among NCK default values.
```

Comment:
Maybe we should just document somewhere what are these NCK default values (once for all readers) ?

Reply:
Yes you are right. For now, as it is implemented in only 1 reader, I've added them directly in the doc of this first reader. Once the feature extended to other readers, we will refactor the doc.